gateway-ha/pom.xml

```diff
@@ -568,6 +568,21 @@
                     </dependency>
                 </dependencies>
             </plugin>
+
+            <plugin>
```

Comment:
Cant we just inherit that from airbase instead or is that not defined there? Also we need to make sure this adds and does not override other rules.

Reply:
Also .. maybe this should go in pluginMgt in the parent

Reply:
Airbase doesn't have this rule https://github.com/airlift/airbase/blob/f9cb1aafcbe1392d45414ddecab186468e7e573c/airbase/pom.xml#L688-L738

> Also we need to make sure this adds and does not override other rules.

I already confirmed it's append.